examples/video_stream/server.py

```
@@ -0,0 +1,547 @@
+    app.router.add_post("/api/video", handle_video)
+    return app
+
+
+def _parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description="Proxy web server for Dynamo video demos"
+    )
+    parser.add_argument(
+        "--dynamo", default="http://localhost:8000", help="Dynamo HTTP server URL"
+    )
+    parser.add_argument("--host", default="0.0.0.0", help="Bind address")
```

Ruff (0.15.10)

[error] Possible binding to all interfaces

(S104)